doc/SecurityBaseline.xml

```diff
@@ -322,25 +333,87 @@
     <table xml:id="signaturesTable">
       <title>Signature Baseline</title>
       <tgroup cols="3">
-        <colspec colname="c1" colwidth="52*"/>
+        <colspec colname="c1" colwidth="15*"/>
+        <colspec colname="c2" colwidth="12*"/>
         <thead>
           <row>
             <entry>
               <para>Name</para>
             </entry>
+            <entry>
+              <para>OID</para>
+            </entry>
             <entry>
               <para>Reference</para>
             </entry>
           </row>
         </thead>
         <tbody valign="top">
           <row>
-            <entry><para>sha256WithRSAEncryption for RSA keys</para></entry>
-            <entry><para>RFC 4055</para></entry>
+            <entry>
+              <para>sha256WithRSAEncryption</para>
+            </entry>
+            <entry>
+              <para>1.2.840.113549.1.1.11</para>
+            </entry>
+            <entry>
+              <para>RFC 4055</para>
+            </entry>
           </row>
           <row>
-            <entry><para>ECDSA for EC keys</para></entry>
-            <entry><para>RFC 5758, X9.62</para></entry>
+            <entry>
+              <para>sha384WithRSAEncryption</para>
+            </entry>
+            <entry>
+              <para>1.2.840.113549.1.1.12</para>
+            </entry>
+            <entry>
+              <para>RFC 4055</para>
+            </entry>
+          </row>
+          <row>
+            <entry>
+              <para>sha512WithRSAEncryption</para>
+            </entry>
+            <entry>
+              <para>1.2.840.113549.1.1.13</para>
+            </entry>
+            <entry>
+              <para>RFC 4055</para>
+            </entry>
+          </row>
+          <row>
+            <entry>
+              <para>ecdsa-with-SHA256</para>
```

Comment:
as mentioned in earlier Security Base line PR review comment, Section 8, Section 10 needs to be combined both are taking about signature algorithms

Example: ECDSA SHA 256, SHA 384, SHA 512 in section 8 can be split into below 3 parts as listed in section 10. 

ecdsa-with-SHA256
ecdsa-with-SHA384
ecdsa-with-SHA512